### DISCUSSION TITLE
videoviralterkini.top

### DISCUSSION THREAD

**Discussion by IoBlight** [realodix/AdBlockID#1553] [General]
**Sample URL**

`https://videoviralterkini-top.preview-domain.com/`
`https://videoviralterkini.top`

**Penjelasan Lebih Lanjut**

tolong ditelusuri web tersebut. ada kawan saya yang kena phishing/scam, akun Instagram serta Facebook nya ke-hack.
motifnya pelaku DM ke korban katanya ada video tak senonoh yang menyebutkan nama dan foto si korban.

**Comment by realodix** [comment]
> https://videoviralterkini-top.preview-domain.com/
> https://videoviralterkini.top
> tolong ditelusuri web tersebut.

Ini merupakan situs phishing

> ada kawan saya yang kena phishing/scam, akun Instagram serta Facebook nya ke-hack.

Ketika klik salah satu video, seolah-olah harus login dengan akun facebook, tetapi yang sebenarnya terjadi adalah korban ditampilkan dengan halaman palsu. Ketika memasukkan username dan password, itu sama saja korban memberikan username dan passwordnya.

Segera lakukan pemulihan dengan mengganti password
